Bookish.Web/Controllers/HomeController.cs

```diff
@@ -26,5 +27,13 @@ public ActionResult Contact()
 
             return View();
         }
+
+        public ActionResult Catalogue()
+        {
+            ViewBag.Message = "Our selection of books.";
```

Comment:
Why do you need to set this here? I would expect this either to be hardcoded in the cshtml file, or a property of the model if it needed to be configurable 🙂